### ISSUE TITLE
Dans apptop Ajouter plus d'information pour voir l'exemple sur la fonctionnalité

### ISSUE THREAD

**Issue by TTVXStarWake** [wet-boew/cdts-sgdc#66]
Issue moved from GCCode: https://gccode.ssc-spc.gc.ca/iitb-dgiit/nw-ws/sgdc-cdts/issues/130

@StdGit created the issue

https://ssl-templates.services.gc.ca/app/cls/WET/gcweb/v4_0_32/cdts/appTop/refFooter_exitscript-fr.shtml
![image (3)](https://user-images.githubusercontent.com/3074084/73949667-b83ebd80-48c8-11ea-8295-f1185668eba2.png)

**Comment by MaximMartel** [comment]
@shawnthompson commented: 

Should be moved to the [Documentation](https://github.com/cenw-wscoe/sgdc-cdts) project

**Comment by patricehernert** [comment]
@MaximMartel Could you confirm this issue is still valid for version 4.0.39 ? Thanks.

**Comment by StdGit** [comment]
Le problème est toujours valide
---
Issue is still valid

**Comment by ahmad-shahid** [comment]
ExitScript is documented in our normal templates which also implies same functionality for application templates. Closing issue.
